packages/web-components/src/dialog-body/dialog-body.stories.ts

```diff
@@ -0,0 +1,193 @@
+import { html } from '@microsoft/fast-element';
+import type { Args } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import { Dialog as FluentDialog } from '../dialog/dialog.js';
+import type { DialogBody as FluentDialogBody } from './dialog-body.js';
+import './define.js';
+import '../button/define.js';
+import '../text/define.js';
+
+type DialogStoryArgs = Args & FluentDialogBody;
+
+const dismissed20Regular = html<DialogStoryArgs>`
+  <svg
+    fill="currentColor"
+    aria-hidden="true"
+    width="20"
+    height="20"
+    viewBox="0 0 20 20"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="m4.09 4.22.06-.07a.5.5 0 0 1 .63-.06l.07.06L10 9.29l5.15-5.14a.5.5 0 0 1 .63-.06l.07.06c.18.17.2.44.06.63l-.06.07L10.71 10l5.14 5.15c.18.17.2.44.06.63l-.06.07a.5.5 0 0 1-.63.06l-.07-.06L10 10.71l-5.15 5.14a.5.5 0 0 1-.63.06l-.07-.06a.5.5 0 0 1-.06-.63l.06-.07L9.29 10 4.15 4.85a.5.5 0 0 1-.06-.63l.06-.07-.06.07Z"
+      fill="currentColor"
+    ></path>
+  </svg>
+`;
+
+const dismissCircle20Regular = html`<svg
+  fill="currentColor"
+  aria-hidden="true"
+  width="20"
+  height="20"
+  viewBox="0 0 20 20"
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <path
+    d="M10 2a8 8 0 1 1 0 16 8 8 0 0 1 0-16Zm0 1a7 7 0 1 0 0 14 7 7 0 0 0 0-14ZM7.8 7.11l.08.06L10 9.3l2.12-2.12a.5.5 0 0 1 .64-.06l.07.06c.17.18.2.44.06.64l-.06.07L10.7 10l2.12 2.12c.17.17.2.44.06.64l-.06.07a.5.5 0 0 1-.64.06l-.07-.06L10 10.7l-2.12 2.12a.5.5 0 0 1-.64.06l-.07-.06a.5.5 0 0 1-.06-.64l.06-.07L9.3 10 7.17 7.88a.5.5 0 0 1-.06-.64l.06-.07a.5.5 0 0 1 .64-.06Z"
+    fill="currentColor"
+  ></path>
+</svg>`;
+
+const closeDialog = (e: Event, id: string, dismissed: boolean = false) => {
+  const dialog = document.getElementById(id) as FluentDialog;
+  dialog.hide();
+};
```

Comment:
These stories probably don't need this `closeDialog` method or any of the `@click` bindings.